src/T2DMap.cpp

```diff
@@ -3727,11 +3736,170 @@ void T2DMap::slot_setRoomSymbol(QString newSymbol, QColor symbolColor, QSet<TRoo
+    QHash<int, int> usedWeights; // key is weight, value is count of uses
+    QHash<bool, int> usedLockStatus;
+
+    while (itRoom.hasNext()) {
+        TRoom* room = mpMap->mpRoomDB->getRoom(itRoom.next());
+        if (!room) {
+            continue;
+        }
+        roomPtrsSet.insert(room);
+        isAtLeastOneRoom = true;
+
+        // Scan and count all the different names used
+        if (!room->name.isEmpty()) {
+            QString thisName = QString(room->name);
+            if (!thisName.isEmpty()) {
+                if (usedNames.contains(thisName)) {
+                    (usedNames[thisName])++;
+                } else {
+                    usedNames[thisName] = 1;
+                }
+            }
+        }
+
+        // Scan and count all the different room colors used
+        int thisColor = room->environment;
+        if (usedColors.contains(thisColor)) {
+            (usedColors[thisColor])++;
+        } else {
+            usedColors[thisColor] = 1;
+        }
+
+        // Scan and count all the different symbols used
+        QString thisSymbol = QString(room->mSymbol);
+        if (usedSymbols.contains(thisSymbol)) {
+            (usedSymbols[thisSymbol])++;
+        } else {
+            usedSymbols[thisSymbol] = 1;
+        }
+
+        // Scan and count all the different weights used
+        int thisWeight = room->getWeight();
+        if (thisWeight > 0) {
+            if (usedWeights.contains(thisWeight)) {
+                (usedWeights[thisWeight])++;
+            } else {
+                usedWeights[thisWeight] = 1;
+            }
+        }
+
+        // Scan and count all the different lock status used
+        bool thisLockStatus = room->isLocked;
+        if (usedLockStatus.contains(thisLockStatus)) {
+            (usedLockStatus[thisLockStatus])++;
+        } else {
+            usedLockStatus[thisLockStatus] = 1;
+        }
+    }
+
+    // No need to show dialog if no rooms were found
+    if (!isAtLeastOneRoom) {
+        return;
+    }
+
+    mpDlgRoomProperties = new dlgRoomProperties(mpHost, this);
+    mpDlgRoomProperties->init(usedNames, usedColors, usedSymbols, usedWeights, usedLockStatus, roomPtrsSet);
+    mpDlgRoomProperties->show();
+    mpDlgRoomProperties->raise();
+    connect(mpDlgRoomProperties, &dlgRoomProperties::signal_save_symbol, this, &T2DMap::slot_setRoomProperties);
+    connect(mpDlgRoomProperties, &QDialog::finished, this, [=]() {
+        mpDlgRoomProperties = nullptr;
+    });
 }
 
 
+void T2DMap::slot_setRoomProperties(
+    bool changeName, QString newName,
+    bool changeRoomColor, int newRoomColor,
+    bool changeSymbol, QString newSymbol,
+    bool changeSymbolColor, QColor newSymbolColor,
+    bool changeWeight, int newWeight,
+    bool changeLockStatus, bool newLockStatus,
+    QSet<TRoom*> rooms)
+{
+    if (newName.isEmpty()) {
+        newName = QString();
+    } else {
+        // 8.0 is the maximum supported by all the Qt versions (>= 5.7.0) we
```

Comment:
I guess this is copy/pasted from elsewhere, but we're at 5.11 now minimum

Reply:
Yes it is taken from the symbol dialog verbatim and I am not sure if this holds true anymore. May warrant a new issue to look into further down the line.

Reply:
Nope, we are now at **Qt 5.14.0 *minimum***:
![image](https://user-images.githubusercontent.com/6163092/199849738-9b2ed678-126b-479a-854d-6e9819b2f5fe.png)
as such we can now change that to be `QChar::Unicode_10_0`:
![image](https://user-images.githubusercontent.com/6163092/199850155-f68f95c2-97ab-4531-8e92-ad2d53d96c16.png)